<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>82</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Selecione a Cor desta Label pelo Menu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCores">
    <property name="title">
     <string>Cores</string>
    </property>
    <addaction name="actionAzul"/>
    <addaction name="actionVerde"/>
    <addaction name="actionVermelho"/>
   </widget>
   <addaction name="menuCores"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAzul">
   <property name="text">
    <string>Azul</string>
   </property>
  </action>
  <action name="actionVerde">
   <property name="text">
    <string>Verde</string>
   </property>
  </action>
  <action name="actionVermelho">
   <property name="text">
    <string>Vermelho</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
